azure-pipelines.yml

```diff
@@ -12,6 +12,15 @@ jobs:
 - job: build
   displayName: "Build"
 
+  strategy:
+    matrix:
+      Java8:
+        jdk_version: 1.8
+        artifact_suffix: '-Java8'
```

Comment:
I would prefer the suffix to be `" Java 8"` since it appears in the name of the artifact, not the filename

Reply:
It's used in the filename as well. I tried changing it but got an error: https://dev.azure.com/glowstonemc/Glowstone/_build/results?buildId=60&view=logs&jobId=d59b01f9-f6ba-5c44-01e4-578439ff0ce1&taskId=99fccf4a-57f7-54e6-97b7-8a840e9f0e41&lineStart=9&lineEnd=10&colStart=1&colEnd=1

In follow-up PRs, we can try fancier stuff like using different suffixes in different places.